src/precord.c

```diff
@@ -271,7 +271,7 @@ void CleanPRecCopy (
     else {
 
         /* remove the forwarding pointer                                   */
-        SET_RNAM_PREC( rec, 1, GET_RNAM_PREC( GET_RNAM_PREC( rec, 1 ), 1 ) );
+        SET_RNAM_PREC( rec, 1, GET_RNAM_PREC( (Obj)GET_RNAM_PREC( rec, 1 ), 1 ) );
```

Comment:
Just wondering what this is doing?

Reply:
Turning `GET_RNAM_PREC` into a function caused this code to produce a cast warning -- quite correctly so, as this function returns an `UInt`, not an `Obj`. But in this case, it is still the right thing to do, as we are extracting a forwarding pointer (see also e.g. `CopyPRecCopy`).

That said, it would certainly be cleaner if we used `ADDR_OBJ` here, as we do for plists. But I'd prefer to do that in a later PR.

Reply:
See also PR #1642